src/services/apiService.ts

```diff
@@ -3,14 +3,16 @@ import axios, {
   type AxiosRequestConfig,
 } from 'axios'
 import { config } from 'config'
+import { useAxios } from '@kong-ui-public/entities-shared'
 
 const adminApiUrl = config.ADMIN_API_URL
 
 class ApiService {
   instance: AxiosInstance
 
   constructor() {
-    this.instance = axios.create({
+    this.instance = useAxios().axiosInstance || axios.create({
```

Comment:
Hi @exuan, `useAxios` should always return an axios instance, so I don't think we need `|| axios.create(...)` here.